### ISSUE TITLE
UI Bug: Border control is messed up

### ISSUE THREAD

**Issue by W2Wizard** [FlaxEngine/FlaxEngine#368]
**Issue description:**
The border control's edges are pretty much messed up. And have overall these 3 issues:

### 1. The horizontal parts are shifted slightly to the left.
![image](https://user-images.githubusercontent.com/63303990/111758983-6921d500-889d-11eb-8994-7b7bd243a2bf.png)

### 2. The edges have these tiny triangles.
When setting the size above 20~ you can notice triangles forming on the corners
![image](https://user-images.githubusercontent.com/63303990/111759268-b2722480-889d-11eb-8800-1c7537ff2a0c.png)

### 3. The border's transforms have a weird behaviour when moving further or closer from the control.
In this example I set border size to 100 to make it more obvious, but it can be noticed with any size.

https://user-images.githubusercontent.com/63303990/111758603-fd3f6c80-889c-11eb-98e1-fc3bd796b33b.mp4

**Flax version:**
1.0

**Comment by mafiesto4** [comment]
It might be related to `RenderingFeatures.VertexSnapping`. It's used to snap vertices to screen pixels so UI is less blurry and more stable visually. For 3D UI it doesn't work good. Sth like this could help:
```cs
var features = Render2D.Features;
Render2D.Features = Render2D.RenderingFeatures.None;

// draw stuff

Render2D.Features = features;
```
I guess we might want to do it by auto for 3D UI.

**Comment by mafiesto4** [comment]
Fixed in 91672b55d8a7935c2c20d5f17d0fb07787e4338b
